src/main/java/nl/fuchsia/controller/ZaakController.java

```diff
@@ -42,7 +42,7 @@ public Zaak getZaakById(@PathVariable("zaakNr") Integer zaakNr) {
         return zaakService.getZaakById(zaakNr);
     }
 
-    @GetMapping(value = "/persoon/{persoonnr}")
+    @GetMapping(value = "zaken?persoonnr={persoonnr}")
```

Comment:
Ik bedoelde eigenlijk als request parameter, niet als path variable.

Reply:
Weer aangepast, kun je kort uitleggen waarom je deze nu gebruikt ipv de path variable?

Reply:
Het pad zelf gebruik je om naar een specifieke resource te wijzen. _Elk_ onderdeel van het pad moet naar een duidelijke resource wijzen. Dat kan een enkele resource zijn of een groep van meerdere (/zaken/1 of /zaken zijn allebei resources).
Als ik /zaken/persoon doe dan weet ik niet waar dit naar moet verwijzen.
Daarnaast, als je in het pad op een 'groepresource' zit, is de enige logische specificering één entiteit uit die groep.

Nou, als je zo'n groepresource hebt en je wilt zoeken op iets anders dan de primaire identifier (de zaaknr hier) dan zul je moeten gaan _filteren_. En dat doe je met url parameters.